### PR TITLE
Add docs on Images

### PR DESCRIPTION
**Pull request by bendichter**

### CODE REVIEW

File: docs/gallery/general/file.py [hunk 1]
```diff
@@ -387,19 +387,63 @@
...
+
+from PIL import Image
+import requests
+import numpy as np
+
+from pynwb.image import RGBAImage, RGBImage, GrayscaleImage
+from pynwb.base import Images
+
+image_url = 'https://flif.info/example-images/fish.png'
+img = Image.open(requests.get(image_url, stream=True).raw)
```
Comment (oruebel) [inline docs/gallery/general/file.py]: For the purpose of testing I would recommend to just add (or use) an image from the static files from the docs. Having to download an image creates an unnecessary dependency on an external service and requires web access and I don't think downloading the image is important for the tutorial.

File: docs/gallery/general/file.py [hunk 2]
```diff
@@ -411,19 +411,52 @@
...
 # .. note::
 #    The Units table has some predefined optional columns. Please review the documentation for
 #    :py:meth:`~pynwb.file.NWBFile.add_unit` before adding custom columns.
 
+####################
+# Images
+# ------
+#
+# You can store static images within the NWB file as well. These can be images of the subject, the environment, stimuli,
+# or really anything.
```
Comment (oruebel) [inline docs/gallery/general/file.py:426]:
```suggestion
# or really anything.
#
# .. note::
#          All basic image types :py:class:`~pynwb.image.RGBAImage`, py:class:`~pynwb.image.RGBImage`, and 
#          py:class:`~pynwb.image.GrayscaleImage` provide the optional: 1) ``description`` parameter to include a
#          text description about the image and 2) ``resolution`` parameter to specify the *pixels / cm* resolution
#          of the image. 
```

File: docs/gallery/general/file.py [hunk 3]
```diff
@@ -411,19 +411,52 @@
...
+img = Image.open('../../source/logo.png')
+
+# you can store an RGBA image
+rgba_logo = RGBAImage(name='RGBA_logo', data=np.array(img))
+
+# or RGB
+rgb_logo = RGBImage(name='RGB_logo', data=np.array(img.convert('RGB')))
+
+# or Grayscale
+gs_logo = GrayscaleImage(name='Grayscale_logo', data=np.array(img.convert('L')))
```
Comment (oruebel) [inline docs/gallery/general/file.py]:
```suggestion
# or Grayscale. Here with the optional description and resolution specified. 
gs_logo = GrayscaleImage(name='Grayscale_logo', 
                         data=np.array(img.convert('L')),
                         description='Grayscale version of the NWB logo',
                         resolution=35.433071)
```

The image has 90 pixels/inch (dpi) which is roughly 35.433071 pixel/cm

**Review by rly — DISMISSED** [review]

**Review by rly — DISMISSED** [review]

**Review by rly — DISMISSED** [review]

**Review by oruebel — DISMISSED** [review]

**Review by rly — DISMISSED** [review]

**Comment by rly** [comment]
@bendichter this is good to go. Thanks!

**Review by rly — APPROVED** [review]

**Pull request merged**